akka-http-core/src/main/scala/akka/http/impl/engine/ws/Masking.scala

```diff
@@ -51,12 +51,9 @@ private[http] object Masking {
       def onPush(part: FrameEvent, ctx: Context[FrameEvent]): SyncDirective =
         part match {
           case start @ FrameStart(header, data) ⇒
-            if (header.length == 0) ctx.push(part)
-            else {
```

Comment:
Yes, that was found by autobahn. The framing protocol details are the main targets of autobahn.